libs/gltfio/src/AssetLoader.cpp

```diff
@@ -435,86 +437,99 @@ void FAssetLoader::createRenderable(const cgltf_node* node, Entity entity) {
 
 bool FAssetLoader::createPrimitive(const cgltf_primitive* inPrim, Primitive* outPrim,
         const UvMap& uvmap, const char* name) {
+    // Declare sentinels for situations where ResourceLoader needs to generate data.
+    const cgltf_accessor* kGenerateNormals = &mResult->mGenerateNormals;
+    const cgltf_accessor* kGenerateTangents = &mResult->mGenerateTangents;
```

Comment:
`kFoo` is a weird naming convention for local variables

Reply:
Good point, in fact these aliases weren't helping readability at all, I've removed them.